### PR TITLE
Add Threshold node

### PR DESCRIPTION
**Pull request by recrudesce**

### CODE REVIEW

File: integrations/document-threshold.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - name: recrudesce
+      socials:
+        github: recrudesce
+        twitter: recrudesce
+pypi: https://pypi.org/project/haystack-threshold-node/
+repo: https://github.com/recrudesce/haystack_threshold_node
+type: Custom Node
+report_issue: https://github.com/recrudesce/haystack_threshold_node/issues
+---
+# haystack_threshold_node
```
Comment (TuanaCelik) [inline integrations/document-threshold.md:15]: Haystack Threshold Node

File: integrations/document-threshold.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---
+layout: integration
+name: Document Threshold
+description: This component filters documents based on a threshold percentage, ensuring only the documents above the threshold get passed down the pipeline.
```
Comment (TuanaCelik) [inline integrations/document-threshold.md:4]: Is it a confidence score percentage? If yes let's say "Filter documents based on a confidence score threshold"

File: integrations/document-threshold.md (hunk 1 above)
Comment (recrudesce) [reply integrations/document-threshold.md]: Changed.

File: integrations/document-threshold.md (hunk 2 above)
Comment (recrudesce) [reply integrations/document-threshold.md]: Changed.

**Review by TuanaCelik — APPROVED** [review]

**Pull request merged**